Bottom: 10e08560e5193ef84be85a2726e65b905d0e752b
Top:    82ebe91d78333e11b43da17f80083fa6de8e6973
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 21:26:05 +0800

Refresh of vhost_net-batch-update-used

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 90d1035..f6571eb 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -451,7 +451,7 @@ static int vhost_net_tx_get_vq_desc(struct vhost_net *net,
 				  out_num, in_num, NULL, NULL);
 
 	if (r == vq->num && vq->busyloop_timeout) {
-		if (vhost_sock_zcopy(vq->private_data))
+		if (!vhost_sock_zcopy(vq->private_data))
 			vhost_net_signal_used(nvq);
 		preempt_disable();
 		endtime = busy_clock() + vq->busyloop_timeout;
